data/styles/ubuntu/themes/SuruBlack.json

```diff
@@ -0,0 +1,16 @@
+{
+	"fontColor": "#CDCDCD",
```

Comment:
Small issue, but spacing here is wrong

Reply:
fixed this, I had to search for the meaning of LGTM 😆